docs/tutorials/Phonons/Phonon_Dispersion_and_DOS.md

```diff
@@ -0,0 +1,297 @@
+In this tutorial, we will examine the phonon DOS and dispersion curves of rocksalt NaH, and examine how the results may be improved (and run quicker) using continuations.
+
+We will use the `cell` file
+
+*nah.cell*
+```
+%BLOCK LATTICE_ABC
+3.393405 3.393405 3.393405
+60 60 60
+%ENDBLOCK LATTICE_ABC
+
+%BLOCK POSITIONS_FRAC
+Na 0 0 0
+H 0.5 0.5 0.5
+%ENDBLOCK POSITIONS_FRAC
+
+SYMMETRY_GENERATE
+
+%BLOCK SPECIES_POT
+NCP
+%ENDBLOCK SPECIES_POT
+
+PHONON_KPOINT_MP_GRID 4 4 4
+phonon_kpoint_mp_offset 0.125 0.125 0.125
+KPOINTS_MP_GRID 4 4 4
+```
+
+and the `param` file
+
+*nah.param*
+```
+task : PHONON
+basis_precision : fine
+xc_functional : LDA
+opt_strategy : SPEED
+fix_occupancy : TRUE
+elec_method : DM
+phonon_sum_rule : TRUE
+phonon_fine_method : INTERPOLATE
+#continuation : nah.check
+```
+
+The flowchart below outlines the steps and results for getting and continuing the DOS and bandstructure results. Any `.pl` boxes are what to put in the command line, and the other boxes are information to put into the `cell` file. For continuations, do not remove the "non-fine" information - so the lines
+
+```
+PHONON_KPOINT_MP_GRID 4 4 4
+phonon_kpoint_mp_offset 0.125 0.125 0.125
+```
+
+should remains, with the "fine" information added in as specified in the flowchart below. Make sure to uncomment `#continuation : nah.check` when performing a continuation - `phonon_fine_method : INTERPOLATE` simply doesn't do anything if you're not calculating any fine information - it only plays a role during the continuations in this example.  
+
+<!DOCTYPE html>
```

Comment:
This all seems a little excessive for viewing a fixed flowchart image that if people want to zoom in on can open the image in its own tab and zoom in naturally. 

It screams "this will go wrongs at some point and someone won't be able to maintain it" to me as well as simply making the source much harder to read.

Possibly relevant alternatives: 
- https://github.com/squidfunk/mkdocs-material/issues/4096 -> https://blueswen.github.io/mkdocs-glightbox/
- https://facelessuser.github.io/pymdown-extensions/extensions/snippets/